packages/utils/src/lib/report-to-md.integration.test.ts

```diff
@@ -13,15 +13,15 @@ describe('report-to-md', () => {
   });
 
   it('should contain all sections when using the fixture report', () => {
-    const date = new Date(2000, 0, 1, 0);
+    const date = new Date(2021, 8, 15, 0);
```

Comment:
This format is generally more readable:

```suggestion
    const date = new Date('2021-09-15');
```